Automattic/jetpack
# All Projects: Handle LICENSE.txt and SECURITY.md automatically.

- Add full GPL to the CLI generate skeleton Closes #19802
- Add GPL info to all plugin projects
- Add auto-licensing to build step Closes #19801
- Auto-copy security notice file




#### Changes proposed in this Pull Request:

* Adds automation around LICENSE and SECURITY files

#### Jetpack product discussion


p2y3YZ-4v3-p2

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* On the build action, ensure no errors for the LICENSE copying.
* Verify the artifact includes LICENSE.txt and SECURITY.md in every project.
